server/mdm/apple/apple_mdm.go

```diff
@@ -219,14 +219,26 @@ func (d *DEPService) RegisterProfileWithAppleDEPServer(ctx context.Context, team
 	if team != nil {
 		tmID = &team.ID
 	}
+
+	// TODO(mna): I don't think we can use this approach here - to only register
+	// for ABM tokens that have hosts in their default team. That's because this
+	// gets called by the Ensure... setup assistant functions, which are called
+	// BEFORE the cron job retrieves the new hosts from ABM, so even if it
+	// doesn't have hosts now, it might have later and the goal of those Ensure
+	// calls is to have those setup assistants defined in Apple and ready to use
+	// for the hosts when they come in.
```

Comment:
^ I think this might need to change when we tackle the rest of the cron changes, after all.